src/conditional-flow/multiple-conditions.js

```diff
@@ -4,15 +4,27 @@
 function isInRange (num, lower, upper) {
 
   // TODO: write code in this function body to pass the tests
+if (num >= lower && num <= upper) {
+  return true
```

Comment:
This can be simplified `num >= lower && num <= upper` as it will resolve to either a true or false value. Try:
```jsreturn num >= lower && num <= upper```